gdraw/ggdkdraw.c

```diff
@@ -865,6 +887,10 @@ static void _GGDKDraw_DispatchEvent(GdkEvent *event, gpointer data) {
         return;
     } else if ((gw = g_object_get_data(G_OBJECT(w), "GGDKWindow")) == NULL) {
         //Log(LOGDEBUG, "MISSING GW!");
+#ifdef FONTFORGE_CAN_USE_GTK_BRIDGE
+	// If the target window is not GGDKDraw's, pass the event onto GTK
+	gtkb_ProcessEvent(gdisp->gtkb_state, event);
+#endif
```

Comment:
This is the surprisingly simple core of the bridge. Find an event destined for a window you don't know about? Instead of throwing it away, send it up the GTK stack.